pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -308,6 +308,49 @@ func (c *Cloud) ensureLoadBalancerv2(namespacedName types.NamespacedName, loadBa
 			}
 		}
 
+		// Whether the ELB was new or existing, sync attributes regardless. This accounts for things
+		// that cannot be specified at the time of creation and can only be modified after the fact,
+		// e.g. idle connection timeout.
+		if len(loadBalancerAttributes) > 0 {
```

Comment:
It might help this read better if `loadBalancerAttributes` was renamed to something like `desiredLoadBalancerAttributes`.  Reading below,

```go
if  targetValue, ok := loadBalancerAttributes[*foundAttribute.Key]
```

would be easier to understand if it was
```go
if targetValue, ok := desiredLoadBalancerAttributes[*foundAttribute.Key]
```